### PR TITLE
[Docs] Japanese translation of docs/feature_debounce_type.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add a Japanese translation of docs/feature_debounce_type.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_debounce_type.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| anything_else | quantum/debounce/* から他のアルゴリズムを使います | 無し |
+
+**分割キーボードについて**:
+デバウンスコードは分割キーボードと互換性があります。
+
+# 独自のデバウンスコードの使用
+* ```DEBOUNCE_TYPE = custom``` を設定します。
+* ```SRC += debounce.c``` を追加します。
+* 独自の ```debounce.c``` を追加します。例については、```quantum/debounce``` にある現在の実装をみてください。
+* デバウンスは未加工のマトリックススキャンごとに発生します。
```
Comment (mtei) [inline docs/ja/feature_debounce_type.md:35]: かなり意訳になってしまいますが、以下ではどうでしょう。
```suggestion
* 毎回のマトリクススキャンの結果はその度デバウンスによって処理されます。
```

Comment (umi-umi) [reply docs/ja/feature_debounce_type.md]: 訳例ありがとうございます。
rawの訳し方にも悩んでいたのですが、文脈的にはいただいた訳例のほうがずっと分かり易いですね。
ありがとうございます！

**Comment by noroadsleft** [comment]
ありがとうございました！

**Pull request merged**